### ISSUE TITLE
Plantuml ditaa diagram is not visible in refman.pdf

### ISSUE THREAD

**Issue by broucke** [doxygen/doxygen#8657]
**Describe the bug**

The plantuml ditaa diagram is not visible in LaTeX ouput, but it is visible in html output.

**Expected behavior**

I expect the diagram to be visible in `./latex/refman.pdf`.

**To Reproduce**

The example consists out of 2 files: 
- the doxygen configuration file `./doxygen.conf`
- the input file `./mainpage.dox`

``` sh
# Example structure
$ tree
.
├── doxygen.conf
└── mainpage.dox
```

```
/** \mainpage ditaa example
\startuml {example.png} "ditaa example"
ditaa
    +--------+   +-------+    +-------+
    |        | --+ ditaa +--> |       |
    |  Text  |   +-------+    |diagram|
    |Document|   |!magic!|    |       |
    |     {d}|   |       |    |       |
    +---+----+   +-------+    +-------+
        :                         ^
        |       Lots of work      |
        +-------------------------+
\enduml
 */
```

``` sh
# Config file generated with: doxygen -g doxygen.conf

# Manual changes to doxygen.conf:
INPUT=mainpage.dox
PLANTUML_JAR_PATH=/opt/plantuml/plantuml.1.2021.8.jar
```
Commands used to generate the html and LaTeX output:
``` sh
doxygen doxygen.conf && make -C ./latex
```

**Screen shots**
Html output
![html_output](https://user-images.githubusercontent.com/51946272/125081341-3018a500-e0c6-11eb-9606-5dd47efe572d.png)

LaTeX output
![latex_output](https://user-images.githubusercontent.com/51946272/125081349-31e26880-e0c6-11eb-9c51-9d0f351bd492.png)

**Version**

``` sh
# Doxygen compiled from source
$ doxygen --version 
1.9.1 (ef9b20ac7f8a8621fcfc299f8bd0b80422390f4b)
```

``` sh
$ lsb_release -a
No LSB modules are available.
Distributor ID:	LinuxMint
Description:	Linux Mint 19.3 Tricia
Release:	19.3
Codename:	tricia
```

``` sh
$ java -jar /opt/plantuml/plantuml.1.2021.8.jar -version
PlantUML version 1.2021.8 (Sat Jun 26 10:20:59 CEST 2021)
(GPL source distribution)
Java Runtime: OpenJDK Runtime Environment
JVM: OpenJDK 64-Bit Server VM
Default Encoding: UTF-8
Language: en
Country: US
 
PLANTUML_LIMIT_SIZE: 4096

Dot version: dot - graphviz version 2.40.1 (20161225.0304)
Installation seems OK. File generation OK
```

``` sh
$ tex --version
TeX 3.14159265 (TeX Live 2017/Debian)
kpathsea version 6.2.3
Copyright 2017 D.E. Knuth.
There is NO warranty.  Redistribution of this software is
covered by the terms of both the TeX copyright and
the Lesser GNU General Public License.
For more information about these matters, see the file
named COPYING and the TeX source.
Primary author of TeX: D.E. Knuth.
```

**Additional context**
- I also tried this with doxygen 1.8.13, on ubuntu 16.04. (doxygen as binary from distribution). The results are the same.
- It looks like the generated eps file (`./latex/example.eps`) is corrupted somehow. I tried opening it with inkscape and failed. That is probably why the eps to pdf converter is unable to convert it to pdf. The `./latex/example.pdf` file exists, but it is blank. That is why it is probably blank in `./latex/refman.pdf`.
- I could not find any warnings or errors in the `./latex/refman.log` file. The only output related to `./latex/example.pdf` is:

```
(./index.tex <./example.pdf>
<example.pdf, id=25, 597.23125pt x 845.1575pt>
File: example.pdf Graphic file (type pdf)
<use example.pdf>
Package pdftex.def Info: example.pdf  used on input line 3.

```

**Comment by albert-github** [comment]
It looks like the generated `eps` file is not an `eps` file but a `png` file.
It looks like a problem with plantuml, though I will have to check a bit deeper (and see if I can find some older plantuml versions).

**Comment by albert-github** [comment]
I did some further investigations and filed a report with plantuml as: https://forum.plantuml.net/14205/eps-output-not-working-for-ditaa
Hopefully they will fix it quickly.

The bug report at plantuml (for completeness):
```
When I have a plantuml file (ba.pu) like:

@startuml
Bob -> Alice : hello
@enduml

and use:

java -Djava.awt.headless=true -jar plantuml.jar -charset UTF-8 -teps ba.pu

the resulting ba.eps is an eps file, though when I have the file ditaa.pu with:

@startditaa
    +--------+   +-------+    +-------+
    |        | --+ ditaa +--> |       |
    |  Text  |   +-------+    |diagram|
    |Document|   |!magic!|    |       |
    |     {d}|   |       |    |       |
    +---+----+   +-------+    +-------+
        :                         ^
        |       Lots of work      |
        +-------------------------+
@endditaa

and use

java -Djava.awt.headless=true -jar plantuml.jar -charset UTF-8 -teps ditaa.pu

the resulting ditaa.eps is not an eps file but a png file.

Used version:

PlantUML Version 1.2021.8beta3

This problem has been reported against doxygen (https://github.com/doxygen/doxygen/issues/8657), but is clearly not a doxygen problem.
```

**Comment by broucke** [comment]
Ok, makes sense. Thanks for the quick follow up! Hopefully it will be fixed quickly.

**Comment by albert-github** [comment]
I'm not optimistic that it will be fixed soon. On my question on plantuml I got an answer:

![image](https://user-images.githubusercontent.com/5223533/125155850-a5d24e80-e162-11eb-8bbc-9c6af46148f8.png)

(see page: https://plantuml.com/ditaa)

Reopening this issue as an  enhancement request (maybe we also need a line in the documentation for the time being).

**Comment by albert-github** [comment]
I've just pushed a proposed patch, pull request #8661

**Comment by broucke** [comment]
Waw, thanks a lot!

**Comment by albert-github** [comment]
Code has been integrated in master on github (please don't close the issue as this will be done at the moment of an official release).

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.9.2.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
